tests/test_constants.py

```
@@ -0,0 +1,10 @@
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)